pom.xml

```
@@ -65,8 +65,8 @@
   </scm>
 
   <properties>
-    <jenkins.version>1.625.3</jenkins.version>
-    <java.level>7</java.level>
+    <jenkins.version>2.130</jenkins.version>
```

Why not just the next LTS?
```suggestion
    <jenkins.version>2.138.4</jenkins.version>
```

---

Because it was the minimum required to make the tests succeed, but I agree.